config/300-containersource.yaml

```diff
@@ -56,6 +56,8 @@ spec:
               type: string
             serviceAccountName:
               type: string
+            template:
+              type: object
```

Comment:
It'd be nice to let the CRD validating `template`. Is validation done somewhere else?

Reply:
@lionelvillard: What validation are you looking for? The `template` field is optional and if it doesn't exist, it falls back to other fields like `image` etc..

Reply:
We can validate in the webhook like this https://github.com/knative/serving/blob/master/pkg/apis/serving/k8s_validation.go

Reply:
I don't think we should necessarily re-use that validation logic, since it might be more restrictive than what we want for sources.